cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -26,23 +26,26 @@ spec:
         requests:
           storage: {{ .Values.kubecostAggregator.persistentConfigsStorage.storageRequest }}
   - metadata:
-      name: aggregator-storage
+      name: aggregator-db-storage
     spec:
       accessModes: [ "ReadWriteOnce" ]
-      storageClassName: {{ .Values.kubecostAggregator.aggregatorStorage.storageClass }}
+      storageClassName: {{ .Values.kubecostAggregator.aggregatorDbStorage.storageClass }}
       resources:
         requests:
-          storage:  {{ .Values.kubecostAggregator.aggregatorStorage.storageRequest }}
-  {{- if .Values.kubecostAggregator.aggregatorDbStorage }}
+          storage:  {{ .Values.kubecostAggregator.aggregatorDbStorage.storageRequest }}
   - metadata:
-      name: aggregator-db-storage
+      # FIXME: Should Aggregator and cost-model share the persistent-configs PV?
+      # Aggregator being a statefulset makes this weird
+      # In the StatefulSet config, all state should be stored in configmaps
+      # (I think?) not on this specific volume, so its probably safe for each
+      # replica to get its own Volume
+      name: persistent-configs
```

Comment:
is this a duplicate of the first item in this VCT list? the fixme comment makes me think this is intentional? Guess I am having trouble connecting the dots

Reply:
Thanks for pointing this out, I wrote this comment in my initial work when I was still a bit lost in the sauce of the volume complexity. I have a better understanding now -- have a look at the updated comment and let me know what you think.

Reply:
comment much better. Am still wondering about how this interacts with https://github.com/kubecost/cost-analyzer-helm-chart/pull/2886/files#diff-c18be748f5739138335359db3c4d6e28520fe28aead2b3663b19656524645b7eR21

Reply:
sorry - just not feeling the love from github today one these links and PR review stuff. Here is a screenshot of the other persistent configs named VCT I was wondering about 
![image](https://github.com/kubecost/cost-analyzer-helm-chart/assets/6842995/cf82af37-0c8f-49c0-b21f-e62f37a2240d)

Reply:
Oh, yep, definitely a dupe. Removed. Eagle eye!